### DISCUSSION TITLE
Script, Sensor53 l2. Invert led?

### DISCUSSION THREAD

**Discussion by nikito7** [arendst/Tasmota#19952] [Q&A] [answered]
How to invert led?

**Comment by nikito7** [comment]
@gemu2015

**Comment by nikito7** [comment]
Maybe change something here

https://github.com/arendst/Tasmota/blob/ff3e3e9360d9f6702c02e0034110b0910a52ec2d/tasmota/tasmota_xsns_sensor/xsns_53_sml.ino#L1488

**Reply by gemu2015** [reply, marked as answer]
this led indicators where only meant for debugging and inverting the led is not supported.
however you may hack the code if you want an inversion.

**Comment by nikito7** [comment]
Ok thank you
